src/CImath/chalf.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _CHALF_H_
+#define _CHALF_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef struct
+    {
+        unsigned short _h;
+    } imath_half;
+
+    imath_half imath_half_createFromFloat (float f);
```

Comment:
I could suggest the less wordy `imath_half_create` since I think the only thing we can create it from is a float, right?

Reply:
That's true. I'll change that.